hamp_radar/iquick.py

```diff
@@ -73,6 +181,26 @@ def main_blocks(data, o):
 def compact_geometry(
     main_blocks: Iterable[SingleMainBlockGeometry],
 ) -> Iterable[MultiMainBlockGeometry]:
+    """
+    Compacts a sequence of SingleMainBlockGeometry instances into a sequence of
+    MultiMainBlockGeometry instances.
```

Comment:
Sequences are much more specific than iterables. See also [Python ABC Collections](https://docs.python.org/3/library/collections.abc.html#collections-abstract-base-classes).
```suggestion
    Compacts an iterable of SingleMainBlockGeometry instances into an iterable of
    MultiMainBlockGeometry instances.
```